### PR TITLE
Longer instructions for installing app and setting scopes

### PR DESCRIPTION
**Pull request by puzzlement**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,13 +60,49 @@ optional arguments:
...
+messages or files posted by others, you will need to be an [Owner or
+Admin](https://get.slack.help/hc/en-us/articles/218124397-Change-a-member-s-role)
+of the workspace.
+
+#### Deleting messages from public channels
 
 - `channels:history`
 - `channels:read`
-- `chat:write:bot`
+- `chat:write:user
```
Comment (sgratzl) [inline README.md]: please add the missing backtick at the end. also missing in the other `chat:write:user` scopes

Comment (puzzlement) [reply README.md]: Done.

**Comment by puzzlement** [comment]
Changes made.

**Comment by sgratzl** [comment]
thx

**Pull request merged**